Services/Store/StoreManager.php

```diff
@@ -78,8 +82,10 @@ public function getActivateWizzyStores()
                 $storeId = trim($storeConfigs['store_id']);
                 $storeSecret = trim($storeConfigs['store_secret']);
                 $apiKey = trim($storeConfigs['api_key']);
+                $this->storeGeneralConfig->setStore($store->getId());
+                $isSyncEnabled =  $this->storeGeneralConfig->isSyncEnabled();
```

Comment:
Rather this needs to be at the place where we want to deny syncing or adding processes in sync if its not enabled.